solidity/contracts/TransformerOracle.sol

```diff
@@ -4,6 +4,11 @@ pragma solidity >=0.8.7 <0.9.0;
 import './base/BaseOracle.sol';
 import '../interfaces/ITransformerOracle.sol';
 
+/**
+ * @notice This implementation of `ITransformerOracle` assumes that all tokens being transformed only have one underlying token.
+ *         This is true when this implementation was writter, but it may not be true in the future. If that happens, then another
```

Comment:
```suggestion
 *         This is true when this implementation was written, but it may not be true in the future. If that happens, then another
```

Reply:
This has been fixed in a following PR 😊